component-tests/steps/authoriseSteps.ts

```diff
@@ -89,13 +80,19 @@ Given('a valid token with employee id of {string}', function (employeeId: string
     context, undefined, undefined, undefined, undefined, undefined, employeeId);
 });
 
-Given('an employee with id of {string} exists', (employeeId: string) => {
+Given('an employee with id of {string} exists', function (employeeId: string) {
   aws.mock(
     'DynamoDB.DocumentClient', 'get', async params => ({ Item: { staffNumber: employeeId } }));
+
+  const context: AuthoriseStepsContext = this.context;
+  context.restoreDynamoDBDocumentClient = true;
 });
 
-Given('an employee with id of {string} does not exist', (employeeId: string) => {
+Given('an employee with id of {string} does not exist', function (employeeId: string) {
```

Comment:
`function` required here to avoid rebinding `this`?

Reply:
again this was @robshakespeare but it looks like a `this` thing yeah